### ISSUE TITLE
Missing parameters in agent-auth.exe

### ISSUE THREAD

**Issue by BraulioV** [wazuh/wazuh#852]
Hi team,

in the `agent-auth.exe` from Windows, some parameters are missing. While in the Linux version you have all this parameters: 

```shellsession
# /var/ossec/bin/agent-auth -h

Wazuh v3.4.0 - Wazuh Inc. (info@wazuh.com)
http://www.wazuh.com
  agent-auth: -[Vhdt] [-g group] [-D dir] [-m IP address] [-p port] [-A name] [-c ciphers] [-v path] [-x path] [-k path] [-P pass]
    -V          Version and license message
    -h          This help message
    -d          Execute in debug mode. This parameter
                can be specified multiple times
                to increase the debug level.
    -t          Test configuration
    -g <group>  Group to run as (default: ossec)
    -D <dir>    Directory to chroot into (default: /var/ossec)
    -m <addr>   Manager IP address
    -p <port>   Manager port (default: 1515)
    -A <name>   Agent name (default: hostname)
    -c          SSL cipher list (default: HIGH:!ADH:!EXP:!MD5:!RC4:!3DES:!CAMELLIA:@STRENGTH)
    -v <path>   Full path to CA certificate used to verify the server
    -x <path>   Full path to agent certificate
    -k <path>   Full path to agent key
    -P <pass>   Authorization password
    -a          Auto select SSL/TLS method. Default: TLS v1.2 only.
```

In Windows, there are only these parameters available:

```powershell
PS C:\Program Files (x86)\ossec-agent> .\agent-auth.exe -h

Wazuh agent-auth: Connects to the manager to extract the agent key.
Available options:
        -h                  This help message.
        -m <manager ip>     Manager IP Address.
        -p <port>           Manager port (default 1515).
        -A <agent name>     Agent name (default is the hostname).
        -P <pass>           Authorization password.
```

What's wrong?

**Comment by pillarsdotnet** [comment]
See #1452 

Testing branch [dev-agent-auth-win](https://github.com/wazuh/wazuh/tree/dev-agent-auth-win) appears to be on-track for merging in 3.7.

**Comment by vikman90** [comment]
Hi @pillarsdotnet @BraulioV,

PR #1790 closed issue #1452. Please consider closing this issue or updating it for any request.

Best regards.

**Comment by BraulioV** [comment]
Awesome @vikman90! Thanks!
